src/forms/radioButtonGroup.story.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { withKnobs, select, text } from '@kadira/storybook-addon-knobs';
+import { action } from '@kadira/storybook-addon-actions';
+
+import RadioButton from './RadioButton';
+import RadioButtonGroup from './RadioButtonGroup';
+
+storiesOf('RadioButtonGroup', module)
+	.addDecorator(withKnobs)
+	.addWithInfo(
+		'Basic usage',
+		'Renders a group of <RadioButton>s. Demonstrates the basic usage.',
+		() => {
+			const classNameKnob = text('className', '');
+			const directionKnob = select(
+				'Direction',
+				{ row: 'row', column: 'column' },
+				'row'
+			);
+			const selectedValueKnob = select(
+				'Selected',
+				{ one: 'Option 1', two: 'Option 2', three: 'Option 3' },
+				'one'
+			);
+			return (
+				<RadioButtonGroup
```

Comment:
Any advantages of passing `RadioButton` elements as children of `RadioButtonGroup` ass opposed to a prop?

```
<RadioButtonGroup radioButtons={[ <RadioButton ... />, <RadioButton ... /> ]}
```